### PR TITLE
Upd feat 120

### LINKED ISSUE #120: Feature: Changes need in "Let's Get Practical" section
### Type of feature

🍕 Feature

### Current behavior

In the [Let's Get Practical section](https://intro.opensauced.pizza/#/05-how-to-contribute-to-open-source?id=let39s-get-practical), there have been two recurring themes when it comes to people completing the activity:
1. Contributors using the CLI Tool method to submit their profile. 
2. Contributors are not running `npm run contributors:add`and `npm run contributors:generate`commands to generate their profile in the guestbook.

After discussing this with @adiati98 and @BekahHW, we've realized that giving contributors the option to either use the CLI tool or manually adding themselves causes confusion.  

Also, we want to make it clear that contributing to [the pizza-verse repo](https://github.com/open-sauced/pizza-verse) is a good next step to practice contributing to open source. 

### Suggested solution

In the [Getting Started subsection](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md), do the following: 

- [ ] Change the wording in [Step 6](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L147) to this:

```
6. Run `npm run contributors: add` in your terminal. Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices
``` 
- [ ] Delete these lines:
https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L149-L156

- [ ] Delete the instructions in [Step 9](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L217-L228) and replace them with the following statement:
```
9. Run `git log` to check if your changes have been automatically committed. Here’s what you should expect to see as your commit message:

```bash
   docs: add @your_username as a contributor
   ``` Then, press `Q` to run the log.
```

- [ ]  Go to [step 11](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L237C1-L241C7) and renumber it step 10:

```
10. ​​ Push the commit to your forked repository with this command: 
```bash
   git push -u origin branch-name
```

- [ ] To solve the second part of this issue, make the following change to the [last sentence of the exercise](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L245):

`Congratulations on your first contribution! 🎉 If you're ready for your next contribution, head on over to the [pizza-verse repo](https://github.com/open-sauced/pizza-verse) and follow the contribution guidelines for your next contribution. Once you're done, head on over to chapter 6 in this course.`

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Pondy007**
## Description

This PR fixes the feature #120. 

**Solution implemented :**
In the [Getting Started subsection](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md), I did the following:

- [ ] Change the wording in [Step 6](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L147) to this:

> 6. Run `npm run contributors: add` in your terminal. Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choices

 

- [ ] Delete these lines:

```
Lines 149 to 156 in [4577a76](https://github.com/open-sauced/intro/commit/4577a767191b640d3092b61a2c687c864fe96356)

 There are two ways you can complete this activity: 
  
 - Using the CLI tool. 
 - Manually editing the `.all-contributorsrc` file. 
  
 Although adding yourself to the guestbook with the CLI tool is simpler, we recommend manually editing the file to get more experience adding and committing your changes, creating pull requests, and pull request reviews. Below, you'll find the directions for each method: 
  
 **CLI Tool** 
```

- [ ] Delete the instructions in [Step 9](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L217-L228) and replace them with the following statement:

> 9. Run `git log` to check if your changes have been automatically committed. Here's what you should expect to see as your commit message:
> 
> `docs: add @your_username as a contributor`
> 
> Then, press `Q` to run the log.

- [ ] Go to step 11 and renumber it step 10
- [ ] Go to step 12 and renumber it step 11

- [ ] To solve the second part of this issue, I made the following change to the [last sentence of the exercise](https://github.com/open-sauced/intro/blob/4577a767191b640d3092b61a2c687c864fe96356/05-how-to-contribute-to-open-source.md?plain=1#L245):

> Congratulations on your first contribution! 🎉 If you're ready for your next contribution, head on over to the [pizza-verse repo](https://github.com/open-sauced/pizza-verse) and follow the contribution guidelines for your next contribution. Once you're done, head on over to [chapter 6](06-the-secret-sauce.md) in this course.

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [x] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #120 

## Mobile & Desktop Screenshots/Recordings
image
image

## Steps to QA

1. Go to the [Let's get practical](https://github.com/open-sauced/intro/blob/main/05-how-to-contribute-to-open-source.md) section
2. In the Getting Started subsection, check steps 6 to 11
3. Check the last sentence of this section

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?
😊

### CODE REVIEW

File: 05-how-to-contribute-to-open-source.md
Comment (CBID2) [inline 05-how-to-contribute-to-open-source.md:151]: Hi @Pondy007. This part needs to be omitted.

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Pondy007. So far, your PR looks good. You just need to omit a section before I give another review.

File: 05-how-to-contribute-to-open-source.md
Comment (Pondy007) [reply 05-how-to-contribute-to-open-source.md:151]: Ok. Done : line "**Manual Addition**" removed.

File: 05-how-to-contribute-to-open-source.md
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:148]: - Let's add a space to fix the format here.
- Add a period in the last sentence.

```suggestion
6. Run `npm run contributors: add` in your terminal.

   Follow the instructions to add yourself to the guestbook. After you finish and click enter, you must click enter again to confirm your choice.
```

File: 05-how-to-contribute-to-open-source.md
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:201]: docsify supports important and note [helpers](https://docsify.js.org/#/helpers). And we want to start applying these where we see fit.

Please update this to:

```suggestion
   !> If you don't see your profile in the "Contributors" section, zoom out the screen until you can see it before taking a screenshot.
```

File: 05-how-to-contribute-to-open-source.md
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:207]: - Let's move the instruction to close the log up, so it won't be missed.
- Let's change the format of commit message example to code block highlight instead of inline.

 ````
9. Run `git log` to check if your changes have been automatically committed. To close the log, press `Q`.

    Here's what you should expect to see as your commit message:

    ```bash
    docs: add @your_username as a contributor
    ```
````

File: 05-how-to-contribute-to-open-source.md
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:213]: Let's add indentation to fix the format.

````suggestion
   ```bash
   git push -u origin branch-name
   ```
````

File: 05-how-to-contribute-to-open-source.md
Comment (adiati98) [inline 05-how-to-contribute-to-open-source.md:217]: Let's add helper for general tip here.

```suggestion
Congratulations on your first contribution! 🎉

?> If you're ready for your next contribution, head on over to the [pizza-verse repo](https://github.com/open-sauced/pizza-verse) and follow the contribution guidelines for your next contribution. Once you're done, head on over to [chapter 6](06-the-secret-sauce.md) in this course.
```

File: translations/fr/02-qu-est-ce-que-l-open-source.md
Comment (adiati98) [inline translations/fr/02-qu-est-ce-que-l-open-source.md:1]: This issue doesn't require to modify the French translation. That said, please change this back to previous state. 🙂

If you have idea to improve the French translation, please open a new issue for us to triage.

**Review by adiati98 — CHANGES_REQUESTED** [review]
Hey @Pondy007,

First of all, thank you for the PR and the prompt changes! Really appreciate it. ✨

Our intention in this issue is to remove the "Manual Addition" part as we will go for adding contributors with the CLI only.

So, from our side, we apologize that we missed to include an instruction to remove the subsection and clarify the wording adjustment accordingly that cause this kind of long review. 🙏

1. I've updated [the issue](https://github.com/open-sauced/intro/issues/120) to include removing the "Manual Addition" subsection. It's the one that I haven't checked.

2. After you done that, I've also left reviews here for you to fix.

Please let me know if you have questions or any help by tagging me here.

Thank you. 😊

File: translations/fr/02-qu-est-ce-que-l-open-source.md
Comment (Pondy007) [reply translations/fr/02-qu-est-ce-que-l-open-source.md:1]: This concerns another issue (#122) for which I requested a PR. I don't know how it got merged in this PR... the modification is not (should not be) on the same branch. I will try to revert the change on this branch. I am not yet familiar with all commands (like git pull origin main or git push that were mentioned on "Keeping Branches Up to Date" and "Merge conflicts" sections).

Comment (adiati98) [reply translations/fr/02-qu-est-ce-que-l-open-source.md:1]: The easiest way is to copy-paste the file from the `main` branch in this repo, commit, and push it 😉

File: 05-how-to-contribute-to-open-source.md
Comment (Pondy007) [reply 05-how-to-contribute-to-open-source.md:201]: Done

File: 05-how-to-contribute-to-open-source.md
Comment (Pondy007) [reply 05-how-to-contribute-to-open-source.md:148]: Done

File: 05-how-to-contribute-to-open-source.md
Comment (Pondy007) [reply 05-how-to-contribute-to-open-source.md:207]: Done

**Comment by Pondy007** [comment]
Hi,
I made all requested changes. I still did not find how to allow some indentation in VSCode when saving. So I modified through github and it's ok now (and get the file to VSCode with proper indentation). New preview is the following.

image

**Review by adiati98 — APPROVED** [review]
Well done, @Pondy007! 🙌
Thank you for the changes! 😄

For your future reference, once you've fixed something from the suggestions, you can click the resolve button.
Doing this can help you and the maintainer to check which suggestion are done, and which are still need to be fixed.

git merge GIF

**Comment by adiati98** [comment]
> I still did not find how to allow some indentation in VSCode when saving.

This mostly happens because there's a Markdown extension or feature enabled in your VS Code, which is good.

One trick that helps is when you need to indent a line, click tab 2 times, then save it. It will save the correct format of the indentation.

But there are some cases where this trick won't work, for example when creating directory tree in Markdown.
In this case, commit and push the changes without saving it.

Hope this helps 😊

**Review by CBID2 — APPROVED** [review]
Well done @Pondy007! :) Thanks for contributing! :)

**Pull request merged**
